tests/deploy/api/test_e2e.py

```diff
@@ -1,47 +1,40 @@
 import json
 import os
 
-import httpx
 import httpx_sse
 import pytest
+from fastapi.testclient import TestClient
 
-from ragna._utils import timeout_after
 from ragna.core._utils import default_user
 from ragna.deploy import Config
-from tests.utils import ragna_api
+from ragna.deploy._api import app
 
 
-@pytest.mark.parametrize("database", ["memory", "sqlite"])
```

Comment:
Removing the memory test here for two reasons:

1. #320 will remove it from our special cases. Meaning, this is no longer part of our core API, but rather a special case.
2. For some reason, in-memory databases don't play nice with the `TestClient` (tiangolo/fastapi#3906). The proposed solution, i.e. using `poolclass=StaticPool` does indeed work, but will cause the streaming test to crash with encode/starlette#1315. I figured it wasn't worth the effort to dive any deeper than that.